### ISSUE TITLE
content/ api error: "Cannot find 'versions' on Content object, 'versions' is an invalid parameter to prefetch_related()" on 

### ISSUE THREAD

**Issue by alikins** [ansible/galaxy#521]
galaxy version: c37ed19 (also tested with #519 and same results)

when running:

     `ansible-galaxy -s http://localhost:8000 search this_is_a_search_term

Amd making the query:

`GET http://localhost:8000/api/v1/content/?repository__name=content-no-meta&namespace__name=alikins`

server log:

```
2018-05-11 18:22:21,152 DEBUG throttling: RoleDownloadCountThrottle:
2018-05-11 18:22:21,263 INFO basehttp: "GET /api/ HTTP/1.1" 200 95
2018-05-11 18:22:21,267 DEBUG permissions: has_permission returned True
2018-05-11 18:22:21,267 DEBUG throttling: RoleDownloadCountThrottle:
2018-05-11 18:22:21,295 ERROR exception: Internal Server Error: /api/v1/content/
Traceback (most recent call last):
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/views.py", line 494, in dispatch
    response = self.handle_exception(exc)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/views.py", line 454, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/views.py", line 491, in dispatch
    response = handler(request, *args, **kwargs)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/mixins.py", line 42, in list
    page = self.paginate_queryset(queryset)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/generics.py", line 173, in paginate_queryset
    return self.paginator.paginate_queryset(queryset, self.request, view=self)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/pagination.py", line 226, in paginate_queryset
    return list(self.page)
  File "/var/lib/galaxy/venv/lib64/python2.7/_abcoll.py", line 581, in __iter__
    v = self[i]
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/paginator.py", line 145, in __getitem__
    self.object_list = list(self.object_list)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/db/models/query.py", line 250, in __iter__
    self._fetch_all()
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/db/models/query.py", line 1120, in _fetch_all
    self._prefetch_related_objects()
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/db/models/query.py", line 675, in _prefetch_related_objects
    prefetch_related_objects(self._result_cache, *self._prefetch_related_lookups)
  File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/db/models/query.py", line 1458, in prefetch_related_objects
    (through_attr, first_obj.__class__.__name__, lookup.prefetch_through))
AttributeError: Cannot find 'versions' on Content object, 'versions' is an invalid parameter to prefetch_related()
2018-05-11 18:22:21,372 ERROR basehttp: "GET /api/v1/content/?repository__name=content-no-meta&namespace__name=alikins HTTP/1.1" 500 21736
```

**Comment by cutwater** [comment]
Fixed by #522

**Comment by cutwater** [comment]
Fixed by #527
